frontend/ws/chatSocket.js

```diff
@@ -22,7 +22,10 @@ function listenEventsFromServer() {
             }));
         }).
         on('users-online', (userNames) => {
-            console.debug(userNames);
+            store.dispatch(usersListRequestAC({
+                type: 'users-online',
```

Comment:
No need to add this field, since it is not used anyhow. Simple `usersListUpdatedAC(onlineUsers)` will be sufficient.